src/App.js

```diff
@@ -0,0 +1,74 @@
+import { nanoid } from "nanoid";
+import { useState } from "react";
+import { AiFillPlusCircle } from "react-icons/ai";
+import TaskList from "./components/ui/TaskList";
+const App = () => {
+  const [task, setTask] = useState("");
+  const [toDo, setAllToDo] = useState([]);
+
+  const handleOnChange = (event) => {
+    setTask(event.target.value);
+  };
```

Comment:
I think it should be much better to put this handleOnChange function on a single line as we only have a single line of code, this will improve the readability of your code. 
```suggestion
  const handleOnChange = (event) => setTask(event.target.value;
```